README.md

```diff
@@ -1,43 +1,56 @@
 spiderable-longer-timeout
 ====
-
+ - [About package](#about)
+ - [Installation](#installation)
+ - [Setup](#setup)
+ - [Render page](#isreadyforspiderable-boolean)
+ - [Options](#options)
+ - [Set up bot's user agents](#useragentregexps-regexp)
+ - [Cache lifetime (TTL)](#cachelifetimeinminutes-cache-ttl-number)
+ - [Set ignored routes](#ignoredroutes-string)
+ - [Differ Phantomjs from other users](#customquery-booleanstring)
+ - [Supported redirects](#supported-redirects)
+ - [On/Off debug messages](#debug-boolean)
+ - [Enable 404 page and right response](#enable-default-404-response-if-youre-using-iron-router)
+ - [Important notes](#important)
+ - [How to install Phantomjs to server](#install-phantomjs-on-your-server)
+ - [Testing](#testing)
+ - [Test with CURL](#curl)
+ - [Test with Google](#google-tools-fetch-as-google)
+ - [Original Spiderable documentation](#from-meteors-original-spiderable-documentation-see-notes-specific-to-this-branch-above)
+
+### About
 This is a branch of the standard meteor `spiderable` package, with some merged code from
 `ongoworks:spiderable` package. Primarily, this lengthens the timeout to 30 seconds and
 size limit to 10MB. All results will be cached to Mongo collection, by default for 3 hours (180 minutes).
 
-### Install using
+This package will ignore all SSL error in favor of page fetching.
+
+This package supports "real response-code" and "real headers", this means if your route returns `301` response code with some headers - exactly same headers will be returned to whom requested. Also it has support of all kind of [JavaScript redirects](#supported-redirects).
+
+This package has build-in caching mechanism, by default it storing results for 3 hours, to change storing period set `Spiderable.cacheLifetimeInMinutes` to other value in minutes.
+
+### Installation
 ```shell
 meteor add jazeee:spiderable-longer-timeout
 ```
 
-### Required: Do the following to make it work
-#### Set `Meteor.isReadyForSpiderable` property
-Spiderable will wait for a flag to be `true`, which gives finer control while content is preparing to be published.
-
-Set `Meteor.isReadyForSpiderable=true` when Meteor finishes publishing and rendering the UI. See [Guidelines](#guidelines)
-
-#### Optionally set `Spiderable.userAgentRegExps`
-`Spiderable.userAgentRegExps` __{[*RegExp*]}__ - is array of Regular Expressions, of bot user agents that we want to serve statically, but do not obey the `_escaped_fragment_ protocol`.
+#### Setup:
+##### isReadyForSpiderable {*Boolean*}
+On server and client tell to Spiderable what everything is ready. Spiderable will wait for `Meteor.isReadyForSpiderable` to be `true`, which gives finer control while content is preparing to be published.
 ```coffeescript
-Spiderable.userAgentRegExps.push /^vkShare/i
+Router.onAfterAction ->
+  Meteor.isReadyForSpiderable = true if @ready()
```

Comment:
This will likely confuse JavaScript developers. Move if @ready() to line before...